lib/resolve-imports-exports.js

```diff
@@ -0,0 +1,185 @@
+var path = require('path');
+var startsWith = require('string.prototype.startswith');
+
+function makeError(code, message) {
+    var error = new Error(message);
```

Comment:
the stack trace is based on where the error is created; it'd be better to inline all the `new Error` calls.